content/docs/command-reference/exp/init.md

```diff
@@ -1,7 +1,6 @@
 # exp init
 
-Codify project using [DVC metafiles](/doc/user-guide/project-structure) to run
-[experiments](/doc/user-guide/experiment-management).
+Quickly setup any project to use [DVC Experiments].
```

Comment:
I meant this (per last check box in this PR desc.) @dberenbaum @skshetry . Currently `dvc exp init -h` prints `Initialize DVC in the current directory. Expects directory to be a Git repository...` Do we want to update that text?

Reply:
I think that's the `dvc init -h` text 😄 . `dvc exp init -h` says `Initialize DVC experiments`. However, I think we agreed this new language is better, so I'm happy to put in a PR to update in the core repo.

Reply:
> I think that's the dvc init -h text 😄

yep 🤦 sry. OK, thanks for the PR